R/standard_layout.R

```diff
@@ -27,15 +27,15 @@ standard_layout <- function(output,
                             post_output = NULL) {
   # checking arguments
   if (!inherits(output, c("shiny.tag", "shiny.tag.list", "html"))) {
-    stop("output is supposed to be of class shiny.tag or shiny.tag.list")
+    stop("output is supposed to be of class `shiny.tag` or `shiny.tag.list`")
```

Comment:
```suggestion
    stop("output must be of class `shiny.tag` or `shiny.tag.list`")
```

Reply:
this as a good comment but don't you think that this should be checked in checkmate and the error message should be handled there (therefore: will be more consistent across the package and the whole codebase)

Reply:
I have seen `stopifnot` used for argument checks in many places in NEST packages. I think this used to be the standard and a full-on migration to `checkmate` has not been undertaken. I change argument checks to `checkmate` if I happen to do some work in the area. I agree, this is a good opportunity to do that here as well.

Reply:
I have reviewed all `stop()` and `stopifnot()`. Some part of this requires more thinking and testing. I don't want to explode this PR as well as make it more difficult to test just because of that. I picked some low-hanging fruits and change accordingly.